DQMOffline/Trigger/python/HLTEGTnPMonitor_cfi.py

```diff
@@ -0,0 +1,339 @@
+import FWCore.ParameterSet.Config as cms
+
+#this is the config to define t&p based DQM offline monitoring for e/gamma
+
+etBinsStd=cms.vdouble(5,10,15,20,25,30,35,40,45,50,60,80,100,150,200,250,300,350,400)
+scEtaBinsStd = cms.vdouble(-2.5,-2.4,-2.3,-2.2,-2.1,-2.0,-1.9,-1.8,-1.7,-1.566,-1.4442,-1.3,-1.2,-1.1,-1.0,-0.9,-0.8,-0.7,-0.6,-0.5,-0.4,-0.3,-0.2,-0.1,0,0.1,0.2,0.3,0.4,0.5,0.6,0.7,0.8,0.9,1,1.1,1.2,1.3,1.4442,1.566,1.7,1.8,1.9,2.0,2.1,2.2,2.3,2.4,2.5)
+phiBinsStd = cms.vdouble(-3.32,-2.97,-2.62,-2.27,-1.92,-1.57,-1.22,-0.87,-0.52,-0.18,0.18,0.52,0.87,1.22,1.57,1.92,2.27,2.62,2.97,3.32)
+
+etRangeCut= cms.PSet(
+    rangeVar=cms.string("et"),
+    allowedRanges=cms.vstring("0:10000"),
+    )
+ecalBarrelEtaCut=cms.PSet(
+    rangeVar=cms.string("scEta"),
+    allowedRanges=cms.vstring("-1.4442:1.4442")
+    )
+ecalEndcapEtaCut=cms.PSet(
+    rangeVar=cms.string("scEta"),
+    allowedRanges=cms.vstring("-2.5:-1.556","1.556:2.5")
+    )
+ecalBarrelAndEndcapEtaCut = cms.PSet(
+    rangeVar=cms.string("scEta"),
+    allowedRanges=cms.vstring("-1.4442:1.4442","-2.5:-1.556","1.556:2.5"),
+    )
+        
+
+hcalPosEtaCut= cms.PSet(
+    rangeVar=cms.string("scEta"),
+    allowedRanges=cms.vstring("1.3:1.4442","1.556:2.5"),
+    )
+hcalNegEtaCut= cms.PSet(
+    rangeVar=cms.string("scEta"),
+    allowedRanges=cms.vstring("-2.5:-1.556","-1.4442:-1.3"),
+    )
+hcalPhi17Cut = cms.PSet(
+    rangeVar=cms.string("phi"),
+    allowedRanges=cms.vstring("-0.87:-0.52"),
+    )
+
+
+
+tagAndProbeConfigEle27WPTight = cms.PSet(
+    trigEvent = cms.InputTag("hltTriggerSummaryAOD","","HLTX"),
+    objColl = cms.InputTag("gedGsfElectrons"),
+    trigResults = cms.InputTag("TriggerResults","","HLTX"),
```

Comment:
yes this was testing settings that were forgotten to be changed, thanks for the spot